src/util/json_irep.cpp

```diff
@@ -38,10 +35,8 @@ json_objectt json_irept::convert_from_irep(const irept &irep) const
     irep_object["id"]=json_stringt(irep.id_string());
 
   convert_sub_tree("sub", irep.get_sub(), irep_object);
-  convert_named_sub_tree("namedSub", irep.get_named_sub(), irep_object);
 
-  if(include_comments)
-    convert_named_sub_tree("comment", irep.get_comments(), irep_object);
```

Comment:
That's not longer present. May or may not matter to any code consuming the JSON output.

Reply:
removed